hadoop-ozone/tools/src/main/java/org/apache/hadoop/ozone/debug/ReadReplicas.java

```diff
@@ -213,10 +205,11 @@ private void downloadReplicasAndCreateManifest(
         }
         replicasJson.add(replicaJson);
       }
-      blockJson.add(JSON_PROPERTY_BLOCK_REPLICAS, replicasJson);
+      blockJson.set(JSON_PROPERTY_BLOCK_REPLICAS, replicasJson);
       blocks.add(blockJson);
 
-      IOUtils.close(LOG, blockReplicasWithoutChecksum.values());
+      blockReplicasWithoutChecksum.values()
+          .forEach(each -> IOUtils.close(LOG, each));
```

Comment:
This part of the change is unrelated, only due to merge conflict resolution.  Let's keep the original:

```suggestion
      IOUtils.close(LOG, blockReplicasWithoutChecksum.values());
```

Reply:
Done!
Thanks!